docs/policy/rdt.md

```diff
@@ -10,43 +10,116 @@ control groups. Resource allocation is specified on the group level and each
+RDT class. In turn, all processes of the container will be assigned to the RDT
+CLOS  (under `/sys/fs/resctrl`) corresponding the RDT class. CRI-RM will configure
+the CLOSes according to its configuration at startup or whenever the
+configuration changes.
 
-### Dynamic Configuration
+By default there is a direct mapping from Pod QoS classes and RDT classes: the
+containers of the Pod get an RDT class with the same name as it's Qos class
+(Guaranteed, Burstable or Besteffort). However, that can be overridden with the
+rdtclass.cri-resource-manager.intel.com Pod annotation. You can also specify
+RDT classes other than Guaranteed, Burstable or Besteffort. In this case, the
+Pod can only be assigned to these classes with the Pod annotation, though.
 
-CRI utilizes configuration received from
-[`cri-resmgr-agent`](../README.md#cri-resource-manager-node-agent), under the
-key `rdt` in the ConfigMap containing `cri-resmgr` configuration data. The
-configuration specifies a set of 'partitions' each further containing a set of
-'classes'. The configuration can be dynamically updated by editing the
-ConfigMap.
+
+## Configuration
+
+The RDT configuration in CRI-RM is a two-level hierarchy that consisting of
+partitions and classes. It specifies a set of partitions each having a set of
+classes.
+
+### Partitions
 
 Partitions represent a logical grouping of the underlying classes, each
 partition specifying a portion of the available resources (L3/MB) which will be
-shared by the classes under it. L3 allocations between partitions are exclusive,
-i.e. no overlap on the cache ways between partitions is allowed. MB
-allocations do not have this property, i.e. all partitions may have 100% of
-bandwidth, for example.
+shared by the classes under it. Partitions guarantee non-overlapping exclusive
+cache allocation - i.e. no overlap on the cache ways between partitions is
+allowed. However, by technology, MB allocations are not exclusive. Thus, it is
+possible to assign all partitions 100% of memory bandwidth, for example.
+
+### Classes
 
-Classes represent the actual RDT classes that the policies assign containers
-to. The set of RDT classes can be freely specified, but, one must ensure that
-classes required by the active policy are specified, and, that the maxmimum
-number of classes (CLOSes) supported by the underlying system is not exceeded.
+Classes represent the actual RDT classes containers are assigned to. In
+contrast to partitions, cache allocation between classes under a specific
+partition may overlap (and they usually do).
 
-CRI-RM has a built-in default configuration containing three classes
-corresponding to the Pod QOS classes of Kubernetes. These are utilized by the
-`static` policy.
+The set of RDT classes can be freely specified, but, it should be ensured that
+classes corresponding the Pod QoS classes are specified. Also, the maximum
+number of classes (CLOSes) supported by the underlying most not be exceeded.
```

Comment:
thinko+typo: ...supported by the underlying hardware must not be exceeded.

Reply:
fixed